File: t6-BasicJAVA/session1/controlFlowStatements/MultiplicationTable.java
// Use a for loop to print a multiplication table.
import java.util.Scanner;

public class MultiplicationTable {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter a number to display its multiplication table: ");
        int number = sc.nextInt();

        System.out.println("Multiplication Table for "  + number + ":");
        for(int i = 1; i <= 10; i++) {
            System.out.println(number + " x " + i + " = " + (number * i));
        }
        sc.close();
    }
}

File: t8-BasicPython/basic_loops/multiplication_table.py
"""
Program 13: Multiplication Table

This program prints the multiplication table of a number.
"""


def print_multiplication_table(number: int) -> None:
    for multiplier in range(1, 11):
        print(f"{number} x {multiplier} = {number * multiplier}")


number: int = int(input("Enter a number: "))
print_multiplication_table(number)